src/accounts.ts

```diff
@@ -8,3 +8,7 @@ export function getUserEmail(): string | undefined {
 export function setUserEmail(email: string) {
   writeGlobalConfig({ account: { email } });
 }
+
+export function getAuthor(): string {
+  return getUserEmail() || 'Unknown';
```

Comment:
consider not using a sentinal value and just dealing with possible undefined string. 
```suggestion
export function getAuthor(): string | undefined {
  return getUserEmail();
```